src/main/java/uk/gov/hmcts/darts/common/repository/ExternalObjectDirectoryRepository.java

```diff
@@ -134,6 +134,22 @@ List<Integer> findMediaFileIdsIn2StorageLocationsBeforeTime(ObjectRecordStatusEn
                                                                 ExternalLocationTypeEntity location2,
                                                                 OffsetDateTime lastModifiedBefore);
 
+
+    @Query(
+        """
+            SELECT eod FROM ExternalObjectDirectoryEntity eod
+            WHERE eod.status in :statuses
+            AND eod.externalLocationType = :type
+            AND eod.responseCleaned = :responseCleaned
+            AND eod.lastModifiedDateTime < :lastModifiedBefore
```

Comment:
I think we should add an order by clause here - so that the earliest will always be at the top of the list?
`ORDER BY eod.lastModifiedDateTime`.
What do you think?